index.html

```diff
@@ -262,8 +262,8 @@ <h4 class="heading"><i class="fas fa-globe"></i> FREE HOSTING AND SERVER</h4>
                   </div> 
   
                   <div class="paragraph-wrapper">
-                      <p class="paragraph">Oops seems we started it from Bang! But yea, you heard it right.
-                        All your projects would be hosted for free at <strong>NO EXTRA COST!</strong> On BlackTimber-Labs Physical 	Servers. Forget about those heavy charges by AWS , AZURE, GCLOUD etc to host your 	projects.
+                      <p class="paragraph">
+                        All your projects would be hosted for free at <strong>NO EXTRA COST!</strong> On BlackTimber-Labs' Physical 	Servers. Forget about those heavy charges by AWS , AZURE, GCLOUD etc to host your 	projects.
```

Comment:
Don't remove the line "Oops seems we started it from Bang! But yea, you heard it right." it's fine